packages/kbn-content-management-utils/src/types.ts

```diff
@@ -347,7 +347,7 @@ export interface ContentManagementCrudTypes<
   /**
    * Update item params
    */
-  UpdateIn: UpdateIn<ContentType, Attributes, UpdateOptions>;
+  UpdateIn: UpdateIn<ContentType, Partial<Attributes>, UpdateOptions>;
```

Comment:
@elastic/kibana-data-discovery I believe this is the only line under your ownership that needs review. I made this a `Partial` type because updates allow for partial saved objects attributes

Reply:
This LGTM, but @mattkime can you do the final approval since you're most familiar with this?

Reply:
I'll take a look in a bit. At first glance this should be fine but I need to refresh my memory. I started out overly strict with the idea we could loosen things if needed.